ariadne_codegen/generators/client.py

```diff
@@ -63,23 +63,14 @@ def add_method(
         async_: bool = True,
     ):
         """Add method to client."""
-        method_def = (
-            self._generate_async_method(
-                name=name,
-                return_type=return_type,
-                arguments=arguments,
-                arguments_dict=arguments_dict,
-                operation_str=operation_str,
-            )
-            if async_
-            else self._generate_method(
-                name=name,
-                return_type=return_type,
-                arguments=arguments,
-                arguments_dict=arguments_dict,
-                operation_str=operation_str,
-            )
-        )
+        kwargs = {
+            "name": name,
+            "return_type": return_type,
+            "arguments": arguments,
+            "arguments_dict": arguments_dict,
+            "operation_str": operation_str,
+        }
+        method_def = (self._generate_async_method if async_ else self._generate_method)(**kwargs)
```

Comment:
To be honest this code is even harden to follow than one before change.

Reply:
Got it